luarules/gadgets/game_unit_market.lua

```diff
@@ -149,19 +149,37 @@ local function offerUnitForSale(unitID, sale_price, msgFromTeamID)
     end
 end
 
+local t2cons = {"armack", "armacv", "armaca", "armacsub", "corack", "coracv", "coraca", "coracsub"}
```

Comment:
Could this not be made based on unit parameters? I guess it would also catch enginers and decoy coms.

Reply:
catching engineers and decoys sounds fine tho

Reply:
what about legion?